src/main/resources/application.yml.example

```diff
@@ -14,6 +14,6 @@ spring:
     password: example
   jpa:
     hibernate:
-      ddl-auto: create-drop
+      ddl-auto: create-update
```

Comment:
Close! it's update only, omit the `create-`. For context, here are the possible options:

```
none: Disable DDL handling.
validate: Validate the schema, make no changes to the database.
update: Update the schema if necessary.
create: Create the schema and destroy previous data.
create-drop: Create and then destroy the schema at the end of the session.
```

Update will also create the schema if there is none as 'it is necessary' as it doesn't match what it expects.
Create is a destructive operation. They definitely could have considered a different model at hibernate, but it's what we have to work with :).

Reply:
oh - damn, sorry. just changed it without testing. i have removed the prefix: create-